pkg/services/stats/statsimpl/stats.go

```diff
@@ -258,6 +265,10 @@ func (ss *sqlStatsService) GetSystemUserCountStats(ctx context.Context, query *s
 	})
 }
 
+func (ss *sqlStatsService) IsUnifiedAlertingEnabled() bool {
+	return ss.cfg != nil && ss.cfg.UnifiedAlerting.IsEnabled()
```

Comment:
How certain are you that `cfg.UnifiedAlerting` can't be nil too? Do we need a check for that?